### ISSUE TITLE
Terraform 0.12 Compatibility

### ISSUE THREAD

**Issue by cameronattard** [suzuki-shunsuke/go-graylog#103]
Currently, `terraform 0.12checklist` returns the following warning:

```
- [ ] Provider "graylog" may need to be upgraded to a newer version that supports Terraform 0.12. (Supported version information is not available for this provider.)
```

https://www.terraform.io/docs/extend/terraform-0.12-compatibility.html

**Comment by suzuki-shunsuke** [comment]
Thank you for your information!
I will check the issue.

**Comment by suzuki-shunsuke** [comment]
I have released [v2.5.0](https://github.com/suzuki-shunsuke/go-graylog/releases/tag/v2.5.0) .
Please try to use it.

You may have to update the terraform configuration.

https://github.com/suzuki-shunsuke/go-graylog/pull/106/commits/73abe24bb26ce37ce81e474e8ce947f50e85ddb7
